server/storage/rethinkdb_models.go

```diff
@@ -13,7 +13,7 @@ const (
 
 var (
 	// TufFilesRethinkTable is the table definition of notary server's TUF metadata files
```

Comment:
Cool, thanks!  You can just also edit the file, commit it on this same
branch, and push the branch up to GitHub. The PR should update itself, and
you wouldn't have to open a new one. :)

On Wednesday, June 8, 2016, Avi Vaid <notifications@github.com> wrote:

> In server/storage/rethinkdb_models.go
> <https://github.com/docker/notary/pull/773#discussion_r66372250>:
>
> > @@ -13,7 +13,7 @@ const (
> >
> >  var (
> >  	// TufFilesRethinkTable is the table definition of notary server's TUF metadata files
>
> No issues, I'll send in a fresh PR with this edited. My bad, I completely
> missed it as well
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/notary/pull/773/files/07265a36352de0381a55ec6ebd5b92cf4fc6a802#r66372250>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANn4FndNFQUjjqCkQ8-FS1B6pAg81f5ks5qJ3KEgaJpZM4Ixh5V>
> .
>